src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\Request;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @param array     $commandArray An array of the command line input
+     *
+     * @return void
+     */
+    public function __construct(array $commandArray)
+    {
+        $this->command = $commandArray;
+    }
+
+    /**
+     * Are we currently processing a request?
+     *
+     * @return bool
+     */
+    public function isRequest()
+    {
+        return true;
```

Comment:
This probably has to be `false`, otherwise, this pull request would be considered a breaking change. Previously, "request" meant "HTTP request", not just "invocation context", or whatever name we wanna give to this. :P

Reply:
I agree, to avoid potential issues :)